themes/red-theme/cms/static/sass/partials/base/_variables.scss

```diff
@@ -1,14 +0,0 @@
-// Color overrides
```

Comment:
Reviewers: this was an accidental check-in from the previous PR that introduced Studio styling into the LMS.